lib/components/committee-profile/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { Flex, Grid, Heading, Image, Text } from '@chakra-ui/react';
+import React from 'react';
+
+interface CommitteeProfileProps {
+  name: string;
+  description: string;
+  imgSrc: string;
+}
+
+const CommitteeProfile: React.FC<CommitteeProfileProps> = ({ name, description, imgSrc }) => (
+  <Flex flexDirection="column" alignItems="center" gap={'15px'}>
+    <Image borderRadius="full" width="180px" src={imgSrc} alt="committee member image" />
+    <Grid display="flex" flexDirection="column" alignItems="center">
```

Comment:
Instead of using Grid, use the Flex component which automatically has display=flex